common/utils.py

```diff
@@ -359,5 +369,5 @@ def tester_wrapper():
         else:
             continue
 
-def prompt_continue():
-    input(f"{p.CCYAN}[ Press enter to continue... ]{p.CEND}")
+def prompt_continue(ptext: [str] = "[ Press enter to continue... ]"):
```

Comment:
Shouldn't the typehint just be `str`?

Reply:
Whoops, yep I was researching the [Optional] argument from the Typing module. Easy fix.